endpoints/bookstore-grpc/bookstore_pb2_grpc.py

```diff
@@ -0,0 +1,170 @@
+# Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
```

Comment:
do we need to checkin this file?
I think it's better to update instructions that the user will generate the file rather than checkin autogenerated code.

Reply:
This file was checked-in before.
It is checked-in in all gRPC examples, as far as I can tell.

Reply:
I want to change it so autogenerated files will not be checked in - everywhere. However I'm fine to do it in stages - first unblock this sample as-is and then update all the places to not include autogenerated files (separate checkin).